src/components/common/Loader/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { ActivityIndicator, ActivityIndicatorProps } from 'react-native';
+
+interface LoaderProps extends ActivityIndicatorProps {}
+
+const Loader: React.FC<LoaderProps> = (props) => {
+    return (
+        <ActivityIndicator/>
+    );
+}
```

Comment:
Is there a reason we're creating a Loader component? It doesn't look like we're adding anything on top of ActivityIndicator. Unless I'm missing something here.